microsoft-edge/devtools-guide-chromium/javascript/publish-source-maps-to-azure.md

```diff
@@ -190,3 +190,4 @@ Note here that the `-n` parameter must be unique. Repetition of job names, even
 
 * [Securely debug original code by using Azure Artifacts symbol server source maps](consume-source-maps-from-azure.md)
 * [Map the processed code to your original source code, for debugging](source-maps.md)
+* [Improvements for connecting DevTools to Azure Artifacts symbol server](../whats-new/2023/07/devtools-115.md#improvements-for-connecting-devtools-to-azure-artifacts-symbol-server) in _What's New in DevTools (Microsoft Edge 115)_.
```

Comment:
@captainbrosset - maybe update article per What's New section, since PAT's no longer the only way

Reply:
entered work item for that